abi-check-core/src/test/java/com/palantir/abi/checker/integration/MethodConflictCheckerIntegrationTest.java

```diff
@@ -980,8 +981,7 @@ public static void staticMethod() {
         assertNoConflicts(tempDir);
     }
 
-    // TODO(aldexis): Do not inherit static methods from interfaces
-    @Disabled("This isn't actually working yet")
+    @Disabled("https://github.com/palantir/gradle-transitive-abi-checker/issues/9")
```

Comment:
Also pointing this to https://github.com/palantir/gradle-transitive-abi-checker/issues/9